README.md

```diff
@@ -67,7 +67,7 @@ section of the **Knot.x** documentation.
   src="https://github.com/Cognifide/knotx/blob/master/documentation/src/main/wiki/assets/knotx-intro-prototyping.png?raw=true"
   alt="Knot.x Prototyping"/>
 
-A potential client asked you to prepare a demo presenting the capabilities of a new site. The client
+A potential client asks you to prepare a demo presenting capabilities of a new site. The client
```

Comment:
_A potential client asks you to prepare a demo presenting **the** capabilities of a new site._

You shouldn't have removed **the**